src/cart/crt_corpc.c

```diff
@@ -139,12 +139,13 @@ crt_corpc_initiate(struct crt_rpc_priv *rpc_priv)
 		grp_priv = grp_gdata->gg_srv_pri_grp;
 		D_ASSERT(grp_priv != NULL);
 	} else {
-		grp_priv = crt_grp_lookup_int_grpid(co_hdr->coh_int_grpid);
+		grp_priv = crt_grp_lookup_locked(co_hdr->coh_grpid);
```

Comment:
this call needs to be surrounded by lock/unlock, similar to how crt_grp_lookup_int_grpid() is implemented.

D_RWLOCK_RDLOCK(&crt_grp_list_rwlock);
grp_priv = ...lookup_locked()
if (grp_pri) addref
D_RWLOCK_UNLOCK(&crt_grp_list_rwlock);

Reply:
ok